chapters/planning-data-work.tex

```diff
@@ -1,147 +1,465 @@
-%------------------------------------------------
+% ----------------------------------------------------------------------------------------------
 
 \begin{fullwidth}
-Preparation for data work begins long before you collect any data.
-In order to be prepared to work on the data you receive, 
-you need to know what you are getting into.
+Preparation for collaborative data work begins long before you collect any data,
+and involves planning both the software tools you will use yourself
+and the collaboration platforms and processes for your team.
+In order to be prepared to work on the data you receive with a group,
+you need to plan out the structure of your workflow in advance.
 This means knowing which data sets and output you need at the end of the process,
-how they will stay organized and linked,
-what different types and levels of data you'll handle,
-and whether the data will require special handling due to volume or privacy considerations.
-Identifying these details creates a \textbf{data map} for your project,
+how they will stay organized, what types of data you'll handle,
+and whether the data will require special handling due to size or privacy considerations.
+Identifying these details should help you map out the data needs for your project,
 giving you and your team a sense of how information resources should be organized.
 It's okay to update this map once the project is underway --
 the point is that everyone knows what the plan is.
```

Comment:
```suggestion
the point is that everyone knows - at any given time - what the plan is.
```
This was suggested in #181. Any particular reason it was not implemented or just lost in the maelstrom of issues?